tufoinnkuo10/TODO
# Test add remove part 2

In this part of the project :
- we followed the AAA format
- imported functions to be tested
- used barbel to enable the import and export
- wrote their description for the add and remove test functions
- used jest coverage to test it
- added the mock and dom
- used the jsdom environment to check the dom
- tested edit task, update complete task and clear all task
- update read me and add a screenshot of all tests being succesful
- Add procedure to get it running
